freecad/plot/plotSeries/TaskPanel.py

```diff
@@ -142,77 +142,77 @@ def widget(self, class_id, name):
         name -- Name of the widget
         """
         mw = self.getMainWindow()
-        form = mw.findChild(QtGui.QWidget, "TaskPanel")
+        form = mw.findChild(QtGui.QWidget, "TaskPanel_plotAxes")

```

Comment:
Replace `TaskPanel_plotAxes` by `TaskPanel_plotSeries`

Reply:
Pfff, how I didn't noticed?

Reply:
Hahaha, my PRs are always full of this kind of errors...